azure-pipelines.yml

```diff
@@ -21,17 +21,24 @@ stages:
               CC: gcc-7
               CXX: g++-7
               TEST_X86: 1
-            Linux (gcc 9):
-              vmImage: "ubuntu-latest"
-              CC: gcc-9
-              CXX: g++-9
-            Linux (clang 9):
-              vmImage: "ubuntu-latest"
-              CC: clang-9
-              CXX: clang++-9
+            Linux (gcc 10):
+              vmImage: "ubuntu-20.04"
+              CC: gcc-10
+              CXX: g++-10
+              # GCC does not respect the `-Wvariadic-macros` suppression right now
+              # ERROR_ON_WARNINGS: 1
```

Comment:
How would you feel if we stay on GCC 9 until this is resolved?

Reply:
If I remember correctly, it had the same problems, lol.